scripts/src/internal/services.ts

```diff
@@ -16,7 +16,7 @@ import { AssetUnavailable, BlockchainError, WrongAmount, WrongRecipient, WrongSe
 import { setFailedOrder } from "../public/services/orders";
 import { Application, AppOffer } from "../models/applications";
 
-const BLOCKCHAIN = "stellar-testnet";
+const BLOCKCHAIN = "kin-prod";
```

Comment:
I also accidentally made this change here too. But I think we should keep it instead of adding a new PR